### PR TITLE
Fix lint and make clean

### PR DESCRIPTION
**Pull request by chantra**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -50,7 +50,7 @@ publish:
 # Cleans up python dist files
 .PHONY: clean
 clean:
-	rm -rf atc/atcd/dist atc/atc_thrift/dist atc/django-atc-api/dist atc/django-atc-demo-ui/dist atc/django-atc-profile-storage/dist
+	rm -rf atc/atcd/{build,dist} atc/atc_thrift/{build,dist} atc/django-atc-api/{build,dist} atc/django-atc-demo-ui/{build,dist} atc/django-atc-profile-storage/{build,dist}
```
Comment (zfjagann) [inline Makefile]: rm -rf atc/{atcd,atc_thrift,django-atc-api,django-atc-demo-ui,django-atc-profile-storage}/{build,dist}

Comment (zfjagann) [inline Makefile]: or something like

    rm -rf $(find atc -type d -depth 2 -name dist -o -name build)

**Comment by zfjagann** [comment]
LGTM, land at will. ;)

File: Makefile (hunk 1 above)
Comment (chantra) [inline Makefile]: that would be better. Went for the bash expansion verison

**Comment by chantra** [comment]
the build seems to have failed.... travis seems to have a bad time today so I cant access the logs.

**Pull request merged**